rpc/client/types.go

```diff
@@ -0,0 +1,13 @@
+package client
+
+// ABCIQueryOptions can be used to provide options for ABCIQuery call other
+// than the DefaultABCIQueryOptions.
+type ABCIQueryOptions struct {
+	Height uint64
+	Prove  bool
```

Comment:
Since we're moving to `Prove: true` as the default, maybe switching this to `Trusted bool`, where `Trusted == !Prove` would be better, so that an empty `ABCIQueryOptions{}` would have the expected defaults.